src/ansys/mapdl/core/_commands/database/selecting.py

```diff
@@ -20,1853 +20,2021 @@
+
+            * ``A`` - Add labels to the current set.
+
+            * ``U`` - Unselect (remove) labels from the current set.
+
+            * ``ALL`` - Restore the full set of labels.
+
+            * ``STAT`` - Display the current select status.
+
+        dof1 : str
+            Used only with ``Type`` = S, A, or U. Valid lables are:
+
+            **Structural labels** : UX, UY, or UZ (displacements); U (UX, UY, and UZ) ; ROTX, ROTY, or ROTZ
+            (rotations); ROT (ROTX,
+            ROTY, and ROTZ); DISP (U and ROT); HDSP (Hydrostatic pressure).
+            **Thermal labels** : TEMP, TBOT, TE2, TE3...., TTOP (temperature).
+            **Acoustic labels** : PRES (pressure); UX, UY, or UZ (displacements for FSI coupled elements).
+            **Electric labels** : VOLT (voltage); EMF (electromotive force drop); CURR (current).
+            **Magnetic labels** : MAG (scalar magnetic potential); AZ (vector magnetic potential); A (AZ); CURR
+            (current).
+            **Structural force labels** : FX, FY, or FZ (forces); F (FX, FY, and FZ); MX, MY, or MZ (moments); M
+            (MX, MY, and MZ); FORC (F
+            and M); DVOL (fluid mass flow rate).
+            **Thermal force labels** : HEAT, HBOT, HE2, HE3...., HTOP (heat flow).
+            **Fluid flow force label** : FLOW (fluid flow).
+            **Electric force labels** : AMPS (current flow); CHRG (electric charge).
+            **Magnetic force labels** : FLUX (scalar magnetic flux); CSGZ (magnetic current segment).
+            **Diffusion labels** : CONC (concentration); RATE (diffusion flow rate).
+
+        dof2 : str
+            Used only with ``Type`` = S, A, or U. Valid lables are:
+
+            **Structural labels** : UX, UY, or UZ (displacements); U (UX, UY, and UZ) ; ROTX, ROTY, or ROTZ
+            (rotations); ROT (ROTX,
+            ROTY, and ROTZ); DISP (U and ROT); HDSP (Hydrostatic pressure).
+            **Thermal labels** : TEMP, TBOT, TE2, TE3...., TTOP (temperature).
+            **Acoustic labels** : PRES (pressure); UX, UY, or UZ (displacements for FSI coupled elements).
+            **Electric labels** : VOLT (voltage); EMF (electromotive force drop); CURR (current).
+            **Magnetic labels** : MAG (scalar magnetic potential); AZ (vector magnetic potential); A (AZ); CURR
+            (current).
+            **Structural force labels** : FX, FY, or FZ (forces); F (FX, FY, and FZ); MX, MY, or MZ (moments); M
+            (MX, MY, and MZ); FORC (F
+            and M); DVOL (fluid mass flow rate).
+            **Thermal force labels** : HEAT, HBOT, HE2, HE3...., HTOP (heat flow).
+            **Fluid flow force label** : FLOW (fluid flow).
+            **Electric force labels** : AMPS (current flow); CHRG (electric charge).
+            **Magnetic force labels** : FLUX (scalar magnetic flux); CSGZ (magnetic current segment).
+            **Diffusion labels** : CONC (concentration); RATE (diffusion flow rate).
```

Comment:
We are repeating the same content as `dof1`... what did we agree about this? To repeat or mention same as `argX`?

Reply:
From my remembering, this is what we agreed. Are you fine with it?